### PR TITLE
Bug 1207928 - Stopping event propagation before showing the context menu r=etienne

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=313b1f018fd0ac45c502274136598bbe0305fa9f

File: apps/system/js/browser_context_menu.js [hunk 1]
```diff
@@ -56,6 +56,9 @@
         return;
       }
 
+      evt.preventDefault();
+      evt.stopPropagation();
+
```
Comment (etiennesegonzac) [inline apps/system/js/browser_context_menu.js]: can you add a small unit test?
we already have the plumbing in place to assert on `defaultPrevented` and we can to the same for `stopPropagation`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e437617d28269c5836fd02438099791344ebf09c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e05aa4c1e38e06b87cb2c6b8b1a96159e8e8bed2

**Pull request merged**